docs/source/index.rst

```diff
@@ -241,6 +241,7 @@ Contents
    notebooks/09_RQAOA_example.ipynb
    notebooks/10_workflows_on_Amazon_braket.ipynb
    notebooks/11_Mixer_example.ipynb
+   notebooks/12_testing_azure.ipynb
```

Comment:
should testing_azure be an example notebook?

Reply:
No, probably not. But we will need to revamp all the notebooks anyway. So we can keep it there for the moment (provided that is re-run and the warnings are removed)